cygnus-ngsi/src/main/java/com/telefonica/iot/cygnus/sinks/NGSICartoDBSink.java

```diff
@@ -618,18 +619,18 @@ protected String buildTableName(String servicePath, String entity, String attrib
                             + "dm-by-service-path data model");
                 } // if
 
-                name = NGSIUtils.encodePostgreSQL(servicePath, true);
+                name = NGSICharsets.cartoDBEncode(servicePath.substring(1));
```

Comment:
Yeah, I would vote for encoding the `/` no matter if it is the first character or any other in the service path itself... :) In my opinion, the simpler the rules to encode (and to decode) the better... :)